README.md

```diff
@@ -1 +1,23 @@
 # hw01
+
+## This is my brief Biography!
+
+My name is Eleanor Anderson. I am a current fourth year undergraduate student majoring in Economics and Psychology. I am planning on graduating this quarter and am applying to graduate schools. I wanted to take this class becuase I have had a handful of classes that use R but I never truly learned how R works, I mostly learned how to plug and chug. I am **very** *excited* to learn how to actually use R!
```

Comment:
Great intro! I've had the same experience with classes that use R here, so I'm glad we're in the same boat.

Reply:
Sounds like y'all are in the right place!

Reply:
That's exciting you are graduating this quarter! I'm sure R will also be very useful in learning before your graduate studies. Thanks for sharing your bio, Eleanor.